contrib/kafka/README.md

```diff
@@ -0,0 +1,38 @@
+# KafkaIO : Dataflow Unbounded Source and Sink for Kafka Topics
+
+KafkaIO provides unbounded sources and sinks for [Kafka](https://www.firebase.com/)
+topics. Kafka version 0.9 and above are supported.
```

Comment:
Works with 0.10, but haven't tested with 0.11 (kafka master?). We can remove the line or mention specific versions tested.